src/seedu/addressbook/commands/SortedListCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.Person;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```

Comment:
Should be more descriptive like, Lists all persons in the address book in sorted order to the user.